src/notebook-jobs-panel.tsx

```diff
@@ -63,7 +63,14 @@ export class NotebookJobsPanel extends VDomRenderer<JobsModel> {
 
   showDetailView(jobId: string): void {
     this.model.jobsView = 'JobDetail';
-    this.model.jobDetailModel.jobId = jobId;
+    this.model.jobDetailModel.detailType = 'Job';
+    this.model.jobDetailModel.id = jobId;
+  }
+
+  showJobDefinitionDetail(jobDefId: string): void {
+    this.model.jobsView = 'JobDetail';
```

Comment:
Keep using `JobDetail` here; `JobDefinitionDetail` (defined in `src/model.ts`) is unnecessary and should be deleted.

Or, to avoid ambiguity, rename `JobDetail` to `DetailView` and use it instead for both jobs and job definitions.

Reply:
See above.